### PR TITLE
refactor: stop using notification v1

### PR DESCRIPTION
**Pull request by idoshamun**
Remove all usage of notification v1, it's a prerequisite for deleting the actual tables.

WT-2047

### CODE REVIEW

**Review by DragosIorgulescu — APPROVED** [review]
No comment here 👌

File: __tests__/workers/postUpdated.ts [hunk 1]
```diff
@@ -459,7 +459,11 @@ it('should save a new post with content curation', async () => {
   });
   const posts = await con.getRepository(Post).find();
   expect(posts.length).toEqual(3);
-  expect(posts[2].contentCuration).toStrictEqual(['news', 'story', 'release']);
+  expect(posts[2].contentCuration).toIncludeSameMembers([
```
Comment (rebelchris) [inline __tests__/workers/postUpdated.ts:462]: Leaving it here, but did you mean to remove so many test and test files?

**Review by rebelchris — APPROVED** [review]
Overall no concerns, was just wondering why we deleted the updater workers.
Wouldn't we still want to update notifications in those cases?

File: __tests__/workers/postUpdated.ts (hunk 1 above)
Comment (idoshamun) [reply __tests__/workers/postUpdated.ts:462]: Double checked them and they're relevant only for v1 so yes

**Pull request merged**